src/main/resources/schema.sql

```diff
@@ -0,0 +1,35 @@
+-- courses table
```

Comment:
Do we need this file if we map the classes to the tables in the code?

Reply:
Technically no, since we are using JPA, we do not need this file; though it doesn't hurt to have and could help with customization of the database. May be overkill though.